misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -244,6 +246,11 @@ class TransacterTest {
 
   @Test
   fun `can run consecutive replica reads and transactions`() {
+    // This test only makes sense with Vitess
+    if (!transacter.config().type.isVitess) {
```

Comment:
why does it only make sense with Vitess? We should be able to run replica reads in MySQL as well

Reply:
yeah you're right, there's some vitess specific assertions in there but I can make those conditional